client/src/redux/states.js

```diff
@@ -3,4 +3,7 @@ export const REQUEST_STATE = {
   PENDING: 'PENDING',
```

Comment:
just wondering if `READING` & `WRITING` replace use cases for `PENDING`?

Reply:
ideally, they should, but im going to leave it there for now. i might remove it next pr